packages/apollo-federation/src/service/__tests__/buildFederatedSchema.test.ts

```diff
@@ -141,6 +141,7 @@ type User @key(fields: "id") {
 
       expect(schema.getQueryType()).toMatchInlineSnapshot(`
 type Query {
+  """"""
```

Comment:
This inclusion of a blank comment on the `Query` type in this snapshot from `getQueryType` is a surprising change worth investigating.

Reply:
This was due to us explicitly passing a `description: ''` for the `_entities` type!  I've removed that in b1139b2, though I'm curious what the intent was.

cc @jbaxleyiii @trevor-scheer

Reply:
I'm unsure of the intent there, but since that seems to have previously had no effect and removing it restores previous behavior, your approach in https://github.com/apollographql/apollo-server/commit/b1139b27b3473df129b0a1350f4b2e84b4825b0f looks good to me.

As mentioned in [another commit in this PR](https://github.com/apollographql/apollo-server/pull/3649/commits/1903c081ee02200acca686d99668a4407e19dc1f), this change relates directly to https://github.com/graphql/graphql-js/pull/2177.